ETC.tsv

```diff
@@ -7728,18 +7728,18 @@ ETC_20150317_007728	Pick the herb asked by Cleric Master
 ETC_20150317_007729	Offer worship to the Statue of Goddess Vakarine in Fedimian
 ETC_20150317_007730	Manage the Statue of Goddess Vakarine in Fedimian
 ETC_20150317_007731	Manage the Statue of Goddess Vakarine in Klaipeda
-ETC_20150317_007732	Remove the source of contamination
+ETC_20150317_007732	Remove the source of corruption
 ETC_20150317_007733	Use the Crest of Emptiness towards the girl near the portal
 ETC_20150317_007734	{memo X}Sanctum of Secrecy - Swordsman type_conditions
 ETC_20150317_007735	{memo X}Sanctum of Secrecy - Wizard type_conditions
 ETC_20150317_007736	{memo X}Sanctum of Secrecy - Archer type_conditions
 ETC_20150317_007737	{memo X}Sanctum of Secrecy - Cleric type_conditions
 ETC_20150317_007738	Purpose of the secret box
-ETC_20150317_007739	Passed Magician Owyn quiz
-ETC_20150317_007740	Answer the quiz of Dazzlementer Cordelier 
-ETC_20150317_007741	Answer the quiz of Transmuter Furryode
-ETC_20150317_007742	Collect a sheet from a wizard's book inside the laboratory on Mage's Tower 3rd Floor.
-ETC_20150317_007743	For fail and success
+ETC_20150317_007739	Passed Magician Owyn's quiz
+ETC_20150317_007740	Answer Cordelier's quiz
```

Comment:
In real Mage Tower she has no title. No other magician in there has "Dazzlementer" stuff.

She does have a "Bedazzlement" quest though